.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+node_modules/
+npm-debug.log
```

Comment:
This file should not be here, an npm-debug disappears when you solve the error that non runs into, so any commit with one in it inherently had one or more errors already =)